packages/design/src/colors.css

```diff
@@ -6,13 +6,13 @@
   --color-interactive--hover: var(--color-green--dark);
   --color-destructive: var(--color-red);
   --color-destructive--hover: var(--color-red--dark);
-  --color-interactive--subtle: var(--color-greyBlue);
-  --color-interactive--subtle--hover: var(--color-greyBlue--dark);
+  --color-interactive--subtle: var(--color-blue);
+  --color-interactive--subtle--hover: var(--color-blue--dark);
 
   --color-disabled: var(--color-grey);
   --color-disabled--secondary: var(--color-grey--lighter);
 
-  --color-focus: var(--color-greyBlue--light);
+  --color-focus: var(--color-blue--lighter);
 
   /* Status Colours */
 
```

Comment:
@fagnanm note that the changes to the semantic "surface" and "onSurface" values for statuses are happening at the base layer now, I referenced the values you had been using in Jobber to redefine the "lighter/est" values at that level.